src/adhocracy_core/adhocracy_core/schema/test_init.py

```diff
@@ -113,6 +113,29 @@ def test_sequent_widget_is_fixe_if_readonly(self):
         assert inst.widget.readonly
         assert inst.widget.deserialize(inst, ['default']) == colander.null
 
+class TestSequenceOptionalJsonInSchema:
+
+    def make_one(self, **kwargs):
+        from . import SequenceOptionalJsonInSchema
+
+        class AdhocracySequenceExample(SequenceOptionalJsonInSchema):
+            child1 = colander.Schema(typ=colander.Int())
+        return AdhocracySequenceExample(**kwargs).bind()
+
+    def test_create(self):
+        from . import SequenceOptionalJsonInSchema
+        from . import SchemaNode
+        inst = self.make_one()
+        assert isinstance(inst, SequenceOptionalJsonInSchema)
```

Comment:
The important class inheritance we rely on is not "itself" but the SequenceSchema class